rtt_ros2_services/CMakeLists.txt

```diff
@@ -0,0 +1,103 @@
+cmake_minimum_required(VERSION 3.5)
+project(rtt_ros2_services)
+
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic -Wno-unused-parameter)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_python REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rcpputils REQUIRED)
+find_package(rtt_ros2 REQUIRED)
+find_package(rtt_ros2_node REQUIRED)
+
+# setup Python module
+ament_python_install_package(${PROJECT_NAME}
+  PACKAGE_DIR src/${PROJECT_NAME}
+)
+
+# setup targets
+include_directories(include/orocos)
+
+orocos_library(rtt_ros2_services
+  src/rtt_ros2_services_proxy.cpp
+  src/rtt_ros2_services_registry.cpp
+  EXPORT ${PROJECT_NAME}
+  INCLUDES DESTINATION include/orocos
+)
+target_link_libraries(rtt_ros2_services
+  rtt_ros2_node::rtt_ros2_node
+)
+ament_target_dependencies(rtt_ros2_services rclcpp)
+
+orocos_plugin(rtt_ros2_services_service
+  src/rtt_ros2_services_service.cpp
+)
+target_link_libraries(rtt_ros2_services_service
+  rtt_ros2_services
+)
+ament_target_dependencies(rtt_ros2_services_service rclcpp)
+ament_target_dependencies(rtt_ros2_services_service rcpputils)
+
+orocos_plugin(rtt_ros2_services_registry_service
+  src/rtt_ros2_services_registry_service.cpp
+)
+target_link_libraries(rtt_ros2_services_registry_service
+  rtt_ros2_services
+)
+
+# install
+install(
+  DIRECTORY include/
+  DESTINATION include
+)
+
+# linters
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+  # TODO: check why cppcheck fails on foxy
+  list(APPEND AMENT_LINT_AUTO_EXCLUDE
+    ament_cmake_cppcheck
+  )
```

Comment:
This snippet has been copied from [rtt_ros2_topics/CMakeLists.txt](https://github.com/orocos/rtt_ros2_integration/blob/ab8530f42a62a884ac6dc748525d5c316a6a9a43/rtt_ros2_topics/CMakeLists.txt). For `rtt_ros2_services` the cppcheck test succeeds. Removed in https://github.com/orocos/rtt_ros2_integration/commit/93e3923c97f642ba01046b0e255a9ddf7d375b8e.